server/chat-plugins/hangman.ts

```diff
@@ -113,7 +113,7 @@ export class Hangman extends Rooms.RoomGame {
 	}
 
 	guessWord(word: string, guesser: string) {
-		const ourWord = toID(this.word);
+		const ourWord = toID(this.word.replace(/[^A-Za-z ]/g, ''));
 		const guessedWord = toID(word);
```

Comment:
```suggestion
		const ourWord = toID(this.word.replace(/[0-9]+/g, ''));
		const guessedWord = toID(word.replace(/[0-9]+/g, ''));
```

I understand what your code is doing now, but I think this is more readable (and allows passing in unsanitized guesses for future-proofing).